packages/services/bin/worker.js

```diff
@@ -5,21 +5,22 @@ import errorHandler from '../src/helpers/error-handler'
 import sleep from '@aragon/court-backend-shared/helpers/sleep'
 import Logger from '@aragon/court-backend-shared/helpers/logger'
 
-let [workerPath, name, times, tries, repeat] = process.argv.slice(2)
+let [workerPath, name, times, tries, repeat, prefixColor] = process.argv.slice(2)
 if (!workerPath) throw Error('Cannot start worker with missing path')
 if (name === undefined) name = 'unknown'
 
 times = (times === undefined) ? 1 : parseInt(times)
 tries = (tries === undefined) ? 1 : parseInt(tries)
 repeat = (repeat === undefined) ? 0 : parseInt(repeat)
+prefixColor = (prefixColor === undefined) ? 'white' : prefixColor
 
 Logger.setDefaults(false, true)
 
-const logger = Logger(name)
+const logger = Logger(name, prefixColor)
 const worker = require(path.resolve(process.cwd(), workerPath)).default
 
 async function run() {
-  logger.info(`Starting worker, will run ${times} jobs`)
+  logger.info(`Starting worker, will run ${times == 0 ? 'infinite' : times} jobs`)
```

Comment:
This sounds like a bug... should we change the next line to `let job = 0` instead?

Reply:
Why? I don't get it. I think the idea is if `times == 0`, then the loop will never exit, as `job` cannot be zero, and therefore `times = 0` means infinite. What am I missing?